src/events/utils.py

```diff
@@ -7,11 +7,17 @@
     from events.models import EventPage
 
 
-def get_event_date(event: "EventPage") -> str:
-    return event.event_date.strftime("%-d %B %Y")
+def get_event_start_date(event: "EventPage") -> str:
+    return f"{event.event_start.strftime("%-d %B %Y")}"
 
 
-def get_event_time(event: "EventPage") -> str:
-    start_time = format_time(event.start_time)
-    end_time = format_time(event.end_time)
-    return f"{start_time} - {end_time}"
+def get_event_end_date(event: "EventPage") -> str:
+    return f"{event.event_end.strftime("%-d %B %Y")}"
```

Comment:
Same here as there isn't anything being formatted in a string.
```suggestion
    return event.event_end.strftime("%-d %B %Y")
```